src/components/Loading/loading.scss

```diff
@@ -0,0 +1,8 @@
+@import 'globals';
+
+.root {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 50vh;
```

Comment:
this is weird because when you paginate you get a strange page jump whilst loading.

I'd set a `min-height` and perhaps something smaller?